README.md

```diff
@@ -19,7 +19,7 @@ To add the sbt plugin to your project add the sbt plugin dependency in
 `project/plugins.sbt`:
 
 ```scala
-addSbtPlugin("com.typesafe.play" % "sbt-twirl" % "LATEST_VERSION")
+addSbtPlugin("com.typesafe.sbt" % "sbt-twirl" % "LATEST_VERSION")
```

Comment:
@oscarrobinson Thanks! starting with twirl 1.6 we switch to `com.typesafe.play`, I already released 1.6.0-M1 under that groupId.
What do you think about that:

```suggestion
// twirl 1.6 and newer:
addSbtPlugin("com.typesafe.play" % "sbt-twirl" % "LATEST_VERSION")
// twirl 1.5 and before:
addSbtPlugin("com.typesafe.sbt" % "sbt-twirl" % "1.5.1")
```